src/components/Onboarding/PageIndicator.tsx

```diff
@@ -0,0 +1,98 @@
+import styled from '@emotion/styled';
+import React from 'react';
+import Image from 'next/image';
+import starIcon from '@assets/star.svg';
+import { useRouter } from 'next/router';
+
+interface PageProps {
+  index: Number;
+}
+
+function PageIndicator({ index }: PageProps) {
+  const router = useRouter();
+
+  const handleSkipClick = (e: React.MouseEvent<HTMLButtonElement, MouseEvent>) => {
+    e.preventDefault();
+    router.push('/start');
+  };
+
+  const handleNextClick = (e: React.MouseEvent<HTMLButtonElement, MouseEvent>) => {
+    e.preventDefault();
+    if (index < 2) {
+      let nextArrow = document.getElementsByClassName('slick-next')[0] as HTMLElement;
+      nextArrow.click();
```

Comment:
지금 `updateSlideIndex`를 Carousel에 내려줘서 beforeChange로 주고 있는데요. PageIndicator에 내려주고, 이 함수에서 실행시켜주는 것이 자연스러운 것 같아요. 더불어, prev 도 구현한다고 했을 때, 확장성도 좋은것 같습니다. (prev 필요하지 않을까요? ㅋㅋ 없으니까 어색하네요.

```suggestion
      let nextArrow = document.getElementsByClassName('slick-next')[0] as HTMLElement;
      nextArrow.click();
      updateSlideIndex(idx => idx+1)
```

idx => idx +1 이것은 useState에서 이전 상태값을 받아와서 + 1 해주는 문법입니다. index를 내려줄 필요는 없겠어요!